src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -38,20 +38,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, UserPrefs userPrefs) {
 
         versionedAddressBook = new VersionedAddressBook(addressBook);
         filteredGuests = new FilteredList<>(versionedAddressBook.getPersonList());
-        // Dummy variable for now. Delete when implemented.
-        this.rooms = new UniqueRoomList();
-    }
-
-    public ModelManager(ReadOnlyAddressBook addressBook, UserPrefs userPrefs, UniqueRoomList rooms) {
-        super();
-        requireAllNonNull(addressBook, userPrefs);
-
-        logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
-
-        versionedAddressBook = new VersionedAddressBook(addressBook);
-        filteredGuests = new FilteredList<>(versionedAddressBook.getPersonList());
-        // Dummy variable for now. Delete when implemented.
-        this.rooms = rooms;
+        filteredRooms = new FilteredList<>(versionedAddressBook.getRoomList());
```

Comment:
perhaps someone else can review this, idk if we may need this constructor still

Reply:
I think no need, because now `AddressBook` contains both `Rooms` and `Persons`. To maintain encapsulation, `Room`s and `Person`s should always be in `AddressBook`. This way, when you make an edit to either a room/ person, you will save an entire copy of `AddressBook` again. 

The need to save entire copies of `AddressBook` is because of the undo/ redo feature.

Reply:
Okay! Thanks!